{"name": "Получение заметки по некорректному ID", "status": "passed", "steps": [{"name": "Попытка получения заметки с некорректным ID=123abc", "status": "passed", "start": 1759765432903, "stop": 1759765432904}], "parameters": [{"name": "invalid_id", "value": "'123abc'"}], "start": 1759765432902, "stop": 1759765432905, "uuid": "f9b9ff9d-f338-4dcf-89a8-cee5f1675b6c", "historyId": "944c487232783b87fa638e9774e2867b", "testCaseId": "efeb17e83461f5117735527552790e0e", "fullName": "tests.test_api.TestNoteRetrieval#test_get_note_by_invalid_id", "labels": [{"name": "story", "value": "Получение заметок"}, {"name": "feature", "value": "Управление заметками"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestNoteRetrieval"}, {"name": "host", "value": "E4435"}, {"name": "thread", "value": "8244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}], "titlePath": ["tests", "test_api.py", "TestNoteRetrieval"]}